powermeter/homeassistant.py

```
@@ -55,37 +55,69 @@ def get_json(self, path):
+                return float(val)
+            except (TypeError, ValueError) as e:
+                msg = (
+                    f"Home Assistant sensor {entity} state '{val}' is not numeric"
+                )
+                logger.error(msg)
+                raise ValueError(msg) from e
+        except requests.exceptions.HTTPError as e:
+            if e.response is not None and e.response.status_code == 404:
+                msg = f"Home Assistant sensor {entity} does not exist"
+                logger.error(msg)
+                raise ValueError(msg)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)